### PR TITLE
initial pull request for Peterlubczynski bio

### PR DESCRIPTION
**Pull request by peterlubs**
this is my first pull request
this is for #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,3 +11,5 @@ Please create an Issue in this repo and tag it with the `question` label. (We'll
 - [*Pro Git*](http://git-scm.com/book/en/v2) book
 - [Mastering Markdown](https://guides.github.com/features/mastering-markdown/)
 - [GitHub Flow](https://guides.github.com/introduction/flow/)
+
+I can edit this file too!
```
Comment (peterlubs) [inline README.md:15]: add some stuff here

File: README.md [hunk 2]
```diff
@@ -11,3 +11,5 @@ Please create an Issue in this repo and tag it with the `question` label. (We'll
 - [*Pro Git*](http://git-scm.com/book/en/v2) book
```
Comment (peterlubs) [inline README.md:11]: remember this?

File: README.md (hunk 1 above)
Comment (peterlubs) [inline README.md:15]: note

File: peterlubczynski-bio [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# This is a new file for peterlubczynski-Bio
+## I am a software engineer at intuit
+### this is level 3
+
+# Yeah, I can add more
```
Comment (juryoku) [inline peterlubczynski-bio]: Yea, please do

File: README.md (hunk 1 above)
Comment (ernie-m) [inline README.md:15]: Dude...quit being an overachiever on here!!!! :+1:

**Comment by peterlubs** [comment]
let's ship my crappy changes! :+1: 
:shipit:

**Pull request merged**